testing/junit5-suite/src/main/java/io/helidon/testing/junit5/suite/Suite.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.testing.junit5.suite;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import io.helidon.testing.junit5.suite.spi.SuiteProvider;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * Test suite annotation.
```

Comment:
Is this generic JUnit-related annotation that features no Helidon Data or Helidon Transaction imports part of this PR on purpose? Why?

Reply:
This is something I was discussing with @romain-grecourt for more than a year. It's significant preformance improvement of some integration tests where we need some resources, e.g. docker image, to be available for specific set of junit test classes.